main.md

```diff
@@ -981,6 +981,33 @@ MUST only be used with that server and no other.
 Developers should also take care to not confuse this nonce with the
 OpenID Connect [@OpenID.Core] ID Token nonce, should one also be present.
 
+# Authorization Code Binding to DPoP Key {#dpop_jkt}
+
+Binding the authorization code issued to the client's proof-of-possession key
+can enable end-to-end binding of the entire authorization flow.
+This specification defines the `dpop_jkt` authorization request parameter for this purpose.
+The value of the `dpop_jkt` authorization request parameter is the
+JSON Web Key (JWK) Thumbprint [@!RFC7638] of the proof-of-possession public key.
```

Comment:
need to be explicit about the hash function and string encoding

Reply:
sha256 and base64url I assume but it needs to be stated or somewhat say that it's the same hash and encoding as the jkt cnf

Reply:
It's now explicit, as suggested.  Thanks.